src/main/java/com/dataloom/client/LoomClient.java

```diff
@@ -30,14 +30,14 @@
     public LoomClient( Environment environment, SerializableSupplier<String> jwtToken ) {
         this( () -> {
             final Retrofit retrofit = RetrofitFactory.newClient( environment, jwtToken );
-            return (ApiFactory) retrofit::create;
+            return (ApiFactory) clazz -> retrofit.create( clazz );
```

Comment:
Eclipse marks this as broken, but it's only an issue with ECJ.

Furthermore it hides the fact that it will break cross-JVM (which I haven't had a chance to fix yet).